core/src/com/po/fuck/Camera.java

```diff
@@ -0,0 +1,57 @@
+package com.po.fuck;
+
+import com.badlogic.gdx.graphics.OrthographicCamera;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.Vector2;
+import com.po.fuck.collections.All;
+import com.po.fuck.updates.Updatable;
+import com.badlogic.gdx.utils.ScreenUtils;
+
+import static com.po.fuck.Constants.GAME_HEIGHT;
+import static com.po.fuck.Constants.GAME_WIDTH;
+import static com.po.fuck.Constants.CAMERA_SPEED;
+
+public final class Camera extends CenterDrawer implements Updatable {
+    OrthographicCamera camera = new OrthographicCamera();
+    private Vector2 position = new Vector2();
```

Comment:
Please add a comment explaining the meaning of position. Is it a coordinate of Left-right corner, centre, or something else?

Reply:
Every position in our code is the center, why should this position differ ?